src/common/config_opts.h

```diff
@@ -816,6 +816,8 @@ OPTION(osd_op_thread_suicide_timeout, OPT_INT, 150)
 OPTION(osd_recovery_thread_timeout, OPT_INT, 30)
 OPTION(osd_recovery_thread_suicide_timeout, OPT_INT, 300)
 OPTION(osd_recovery_sleep, OPT_FLOAT, 0.01)         // seconds to sleep between recovery ops
```

Comment:
It's kept around so that the admin can set it with the simple option name (and existing configs won't break).  We do the same with all the other _ssd/_hdd options.